### DISCUSSION TITLE
Table View - Sorting on Hidden Collumns

### DISCUSSION THREAD

**Discussion by jasonmacer** [grafana/grafana#72061] [Dashboards, Panels & Visualizations]
I am running Grafana v10.0.2 on Oracle Linux 8.X, and the system is running well; however, I am having an issue with the sortBy on columns. 

I would assume, and you know what they say about assuming anything, that once the data is pulled I should still be able to sort on those columns, even if they are hidden. That said, if the column is hidden the sortBy feature does not work. 

Am i missing something here, or is there a better plugin other than the "table visualization?

Thanks in advance!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82618. Feel free to continue the discussion around this there. Thank you!
